website/docs/r/main_route_table_association.html.markdown

```diff
@@ -10,6 +10,16 @@ description: |-
 
 Provides a resource for managing the main routing table of a VPC.
 
+It is recommended you **do not** use both `aws_main_route_table_association` to
+manage the default route table **and** use the `aws_default_route_table`,
+due to possible conflict in routes.
+
+For more information about Route Tables, see the AWS Documentation on
+[Route Tables][aws-route-tables].
+
+For more information about managing normal Route Tables in Terraform, see our
+documentation on [aws_route_table][tf-route-tables].
+
```

Comment:
Hi @philnichol 👋  lets add this in a note section and use the same language as the `aws_default_route_table` resource referring back to that page.

```suggestion
~> **NOTE:** **Do not** use both `aws_default_route_table` to manage a default route table **and** `aws_main_route_table_association` with the same VPC due to possible route conflicts. See [aws_default_route_table][tf-default-route-table] documentation for more details.

For more information, see the Amazon VPC User Guide on [Route Tables](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html). For information about managing normal route tables in Terraform, see [`aws_route_table`](/docs/providers/aws/r/route_table.html).
```